### ISSUE TITLE
SSE in 7.5.0 does not work with `@nestjs/platform-fastify`

### ISSUE THREAD

**Issue by KyleLeneau** [nestjs/nest#5695]
## Current behavior
When starting a new project and switching the adapter to use `@nestjs/platform-fastify` the SSE example [here](https://github.com/nestjs/nest/tree/master/sample/28-sse) does not work and results in an error:

```
[Nest] 94277   - 11/13/2020, 12:34:03 AM   [NestFactory] Starting Nest application...
[Nest] 94277   - 11/13/2020, 12:34:03 AM   [InstanceLoader] AppModule dependencies initialized +14ms
[Nest] 94277   - 11/13/2020, 12:34:03 AM   [RoutesResolver] AppController {}: +5ms
[Nest] 94277   - 11/13/2020, 12:34:03 AM   [RouterExplorer] Mapped {, GET} route +2ms
[Nest] 94277   - 11/13/2020, 12:34:03 AM   [RouterExplorer] Mapped {/sse, GET} route +0ms
[Nest] 94277   - 11/13/2020, 12:34:03 AM   [NestApplication] Nest application successfully started +2ms
Application is running on: http://127.0.0.1:3000
[Nest] 94277   - 11/13/2020, 12:34:21 AM   [ExceptionsHandler] destination.write is not a function +18695ms
TypeError: destination.write is not a function
    at SseStream.pipe (/nest/sample/28-sse/node_modules/@nestjs/core/router/sse-stream.js:53:21)
    at RouterResponseController.sse (/nest/sample/28-sse/node_modules/@nestjs/core/router/router-response-controller.js:52:16)
    at /nest/sample/28-sse/node_modules/@nestjs/core/router/router-execution-context.js:167:47
    at /nest/sample/28-sse/node_modules/@nestjs/core/router/router-execution-context.js:47:19
    at processTicksAndRejections (node:internal/process/task_queues:93:5)
    at async Object.<anonymous> (/nest/sample/28-sse/node_modules/@nestjs/core/router/router-proxy.js:9:17)
```

## Input Code
N/A, any sample / new app of the sample linked with `@nestjs/platform-fastify` behaves this way.

## Expected behavior
I would expect that either fastify support is added OR I would expect that the documentation and sample would be updated to call out that it isn't supported yet.

## Possible Solution
I suspect the `SseStream` needs to account what adapter of response type it is applying to and either handle the case or not crash, or crash with a better error that it's not supported at this time.

## Environment

<pre><code>
Nest version: 7.5.0
 
For Tooling issues:
- Node version: v15.0.1
- Platform:  Mac

Others:
- N/A
</code></pre>

**Comment by kamilmysliwiec** [comment]
@soyuka can you look at this issue?

**Comment by kamilmysliwiec** [comment]
Fixed in 7.5.2

**Comment by soyuka** [comment]
thanks @kamilmysliwiec don't hesitate to ping me on sse errors if you don't find the time!
